sections/implementation/frontend-structure.tex

```diff
@@ -81,6 +82,11 @@ \subsection{Components}
 \\\\
 The container for the create user page can be seen on \autoref{create-user}.
 The \texttt{CreateUser} class only contains the state, logic for the functions and a \texttt{render()}.
+Within the state of CreateUser, there is the object firstName, which contains a first name and a boolean to see if it is valid.
+It is instantiated as being true, so that the error messages aren't showing when entering the page.
```

Comment:
```suggestion
It is instantiated with the initial value as true, so that the error messages are not showing when entering the page.
```

måske et billede af fejlen brugeren får